provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitionerrole/PractitionerRoleTransformer.java

```diff
@@ -26,28 +27,35 @@
   String baseUrl;
 
   @Override
-  public PractitionerRole apply(PractitionerRoleWrapper ppms) {
+  public PractitionerRole apply(PractitionerRoleWrapper ppmsData) {
     // TODO organization reference is required
     // location could be populated by caresites
-    ProviderResponse.Value provider = ppms.providerResponse().value().get(0);
-
+    ProviderResponse.Value providerResponse = ppmsData.providerResponse().value().get(0);
+    ProviderContactsResponse.Value providerContacts =
+        ppmsData.providerContactsResponse().value() == null
```

Comment:
I think most of the comments from the other transformer class apply in here as well.